src/loader.js

```diff
@@ -696,116 +685,3 @@
     for (var p in descriptors)
       this[p] = descriptors[p];
   }
-
-  function InternalMapPolyfill() {}
```

Comment:
Sorry @joeldenning... we are not removing the polyfill approach, we're just "hiding" it for now until it is an important implementation concern :)